### PR TITLE
N16: Release

### PR DESCRIPTION
**Pull request by ozkriff**
Preview (https://github.com/rust-gamedev/rust-gamedev.github.io/commit/98617a840105b40ec7d5dfc1435c2cb7c8ff083a): https://rust-gamedev-preview.netlify.app/posts/newsletter-016

Everyone is invited to review!

:timer_clock: Hope to merge this in an hour or so if there're no objections.

_Part of #365 ("Newsletter 16")_

### CODE REVIEW

File: content/posts/newsletter-016/index.md [hunk 1]
```diff
@@ -86,7 +82,7 @@ Listen and Subscribe from the following platforms:
 ## Last Call for [Rust GameDev Survey][survey]
 
 Our annual [Rust Game Development Ecosystem Survey][survey] will be closed
-on Sunday at the end of this week, 6. December 2020.
+on in the end of this week, 11. December 2020.
```
Comment (Lokathor) [inline content/posts/newsletter-016/index.md:85]: "on in" should probably just be "at"

**Review by Lokathor — CHANGES_REQUESTED** [review]

File: content/posts/newsletter-016/index.md (hunk 1 above)
Comment (ozkriff) [reply content/posts/newsletter-016/index.md]: fixed, thanks!

**Review by Lokathor — APPROVED** [review]

**Review by djkoloski — COMMENTED** [review]
A few minor issues. I also noticed that Old Gods wasn't rendering its logo correctly but I wasn't sure if that would be fixed after the merge.

File: content/posts/newsletter-016/index.md [hunk 2]
```diff
@@ -545,29 +541,29 @@ deserialize (rkyv):     16 ns/iter
...
-[vlmutolo]: https://www.reddit.com/r/rust/comments/jx32e8/rkyv_02_and_bytecheck_validation_mutable_archives/gcyfoqc
+[vlmutolo]: https://reddit.com/r/rust/comments/jx32e8/rkyv_02_and_bytecheck_validation_mutable_archives/gcyfoqc
 [toy benchmark]: https://git.sr.ht/~vlmutolo/rkyv-bench/tree/master/src/main.rs
 [architecture and internals of rkyv]: https://davidkoloski.me/blog/rkyv-architecture/
 
 ### [assets_manager] v0.4
 
-[assets_manager] provides an convenient way to work with external files, making
+[assets_manager] provides a convenient way to work with external files, making
 ressources caching and hot-reloading easy and straightforward.
```
[the remark this answers is not in the document]
Comment (ozkriff) [reply content/posts/newsletter-016/index.md]: tnx, fixed!

File: content/posts/newsletter-016/index.md [hunk 3]
```diff
@@ -545,29 +541,29 @@ deserialize (rkyv):     16 ns/iter
...
 
 This month was released version 0.4.0, bringing loads of improvements.
```
[the remark this answers is not in the document]
Comment (ozkriff) [reply content/posts/newsletter-016/index.md]: fixed!

**Review by djkoloski — APPROVED** [review]

**Comment by ozkriff** [comment]
> A few minor issues. I also noticed that Old Gods wasn't rendering its logo correctly but I wasn't sure if that would be fixed after the merge.

Hmmm, it looks fine for me :thinking:

**Comment by djkoloski** [comment]
> 
> 
> > I also noticed that Old Gods wasn't rendering its logo correctly but I wasn't sure if that would be fixed after the merge.
> 
> Hmmm, it looks fine for me 🤔

I think it's just an issue with the rich diff viewer. The URL looks right and should point to the right spot once the branch is merged.

**Comment by ozkriff** [comment]
:timer_clock: merging in ~30 minutes

**Review by AlexEne — APPROVED** [review]
Awesome work!

**Comment by ozkriff** [comment]
The release is quite rushed so feel free to review and report issues even after the merge!

**Pull request merged**